### DISCUSSION TITLE
No sound from iPhone to Google Home Max

### DISCUSSION THREAD

**Discussion by trkel** [philippe44/AirConnect#449] [Q&A] [answered]
First-time user - just installed it this morning following this [procedure](https://richardtech.net/how-to-airplay-google-home-nest-with-raspberry-pi/). 

Question: My Google devices are displayed via AirPlay (<name>+) after setting up AirConnect. When I select them to stream music to, I heard the "connect" sound and can hear the volume "clicker" adjust as I adjust the volume on my phone. However, no music comes through when I try to play it (via Apple Music). Is this expected? Is there something obvious that I am missing? 

Setup / Hardware
- Raspberry Pi (note CPU is at 100%, even with the `-z` flag
- AirConnect Master - Dec 10, 2022
- iPhone iOS 16.2
- Google Home Max (Cast Firmware: 1.56.275994)

**Comment by philippe44** [comment, marked as answer]
Maybe a firewall issue. Best is that you run aircast manually and collect the log. See README

**Reply by trkel** [reply]
Thanks! It wasn't the firewall - it seems to have been a networking issue though. Things I did (there was both an `eth0` and `wlan0` connection - unplugged the `eth0`, restarted the device, and manually stopped the service. I manually ran the binary and it started to work - and then spun it up again via the service.

Thanks! You made one very happy family today.
